blt/settings.py

```diff
@@ -121,8 +122,17 @@
     "whitenoise.middleware.WhiteNoiseMiddleware",
     "tz_detect.middleware.TimezoneMiddleware",
     "blt.middleware.ip_restrict.IPRestrictMiddleware",
+    "corsheaders.middleware.CorsMiddleware",
 )
 
+CORS_ALLOWED_ORIGINS = [
```

Comment:
@DonnieBLT I tried to troubleshoot the CORS policy error which occured during CI/CD Run tests(pull_request). I checked PRs made by others and saw that their PRs were failing test because of the same reason.
I  added this middleware so that cross-origin resource sharing requests could be handled, as server would now handle requests originating from different origins, it's used when frontend and backend are hosted on different domains and they need to communicate.